internal/pack_neon.h

```diff
@@ -243,22 +243,27 @@ class PackingRegisterBlock<
   }
 };
 
-// The format below is only used by one ARM64 kernel.
-// It uses a NEON intrinsic that is only available on ARM64, vpaddq_s16.
-// Would be easy to replace with vpadd_s16 if needed, would be almost as
-// efficient.
-#ifdef GEMMLOWP_NEON_64
```

Comment:
That path was mostly generic ARM 32/64bit code, but it was only used on ARM 64bit and was making use of one intrinsic, vpaddq_s16, that is not available on ARM 32bit. With the new ARM 32bit kernel, we now need this code path on ARM 32bit too, so I:
 - removed the existing #ifdef GEMMLOWP_NEON_64 around this entire path
 - added, inside a new #ifdef GEMMLOWP_NEON_32, a small local implementation of vpaddq_s16 allowing this code path to use this intrinsic even on ARM 32bit where it is not provided by the standard arm_neon.h.

Does that make sense?